doc/source/metadata/Probabilities.rst

```diff
@@ -69,12 +65,14 @@ which runs every hour to generate a 3-member ensemble:
 
 
 Percentiles
-***********
+-----------
 
 This is probably the second most strightforward form,
-as again it still represents actual sets of values of the diagnostic,
-but instead of a set of realizations (consistent over time),
-the set of percentile values represent the values at a set of probability levels.
+as again it still represents actual sets of values of the diagnostic.
+Instead of a set of realizations (consistent over time),
```

Comment:
I'm still not quite sure what "consistent over time" means here in the context of realizations, is this something which could be clarified or elaborated upon, if it's important?

Reply:
Thanks for flagging this up - I've tried to make this clearer by adding to the bracketed text, so it now reads "separate scenarios, each self-consistent over time"